app/helpers/application_helper.rb

```diff
@@ -1214,7 +1215,7 @@ def start_page_allowed?(start_page)
   end
 
   def allowed_filter_db?(db)
-    return false if db.start_with?('Container') && !get_vmdb_config[:product][:containers]
+    return false if (db.nil? || db.start_with?('Container')) && !get_vmdb_config[:product][:containers]
     true
```

Comment:
> ok

@martinpovolny , https://github.com/ManageIQ/manageiq/pull/4075 got merged and this patch is rebased on it, so this code is gone.